src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusAccountAttribute.cs

```diff
@@ -31,5 +31,18 @@ public ServiceBusAccountAttribute(string account)
         /// Gets the name of the ServiceBus connection string to use.
         /// </summary>
         public string Account { get; private set; }
+
+        /// <inheritdoc />
+        string IConnectionProvider.Connection
```

Comment:
I wanted the property name on the various trigger/binding attributes to be named "Connection" which I think is better than "Account". So I implement this explicitly here to avoid confusion on this attribute.